### DISCUSSION TITLE
Is there a reason why `smtpd_sender_restrictions` in /etc/postfix/main.cf has multiple repeated parameters?

### DISCUSSION THREAD

**Discussion by Hashfastr** [docker-mailserver/docker-mailserver#2566] [Q&A] [answered]
For example, `reject_authenticated_sender_login_mismatch` is listed three times under `smtpd_sender_restrictions`:
```
smtpd_sender_restrictions = reject_authenticated_sender_login_mismatch, reject_authenticated_sender_login_mismatch, reject_authenticated_sender_login_mismatch, permit_sasl_authenticated, permit_mynetworks, reject_unknown_sender_domain
```
My understanding is that this has no effect since each it would be equivalent to checking if a variable is true over and over. Is there something more complex at play? I run a custom postfix mail server for my research and want to make sure this isn't something crucial

**Comment by casperklein** [comment, marked as answer]
I cannot confirm this in my setup. Please open an issue for that and post your configuration, so that we can reproduce it.

> I run a custom postfix mail server for my research

What do you mean? Don't you use `docker-mailserver`?

**Reply by Hashfastr** [reply]
I repulled, rm'd the old container, and it seems to be gone now.

> What do you mean? Don't you use `docker-mailserver`?

I use `docker-mailserver` for my personal mail server, but for my research I have built a similar stack and sometimes look at the configuration here for reference.
